tests/lookup/tests.py

```diff
@@ -776,6 +780,30 @@ def test_chain_date_time_lookups(self):
             ordered=False
         )
 
+    def test_comparison_against_decimal_field(self):
```

Comment:
Put in new file `test_decimalfield.py`? Are all the new `Decimal*` lookup classes tested?